dask/dataframe/dispatch.py

```diff
@@ -83,7 +85,33 @@ def tolist(obj):
     return func(obj)
 
 
-def make_meta_util(x, index=None, parent_meta=None):
+def make_meta(x, index=None, parent_meta=pd.DataFrame()):
+    """
+    This method creates meta-data based on the type of ``x``,
+    and ``parent_meta`` if supplied.
+
+    Parameters
+    ----------
+    x : Object of any type.
+        This can be an object of type Dask-like/Frame-like/any arbitrary
+        type.
```

Comment:
Since this can be any type, and this is stated in the line above, I think we can remove the type description here and not loose any pertinent information.

```suggestion
        Object to construct meta-data from.
```